_includes/api-docs.html

```diff
@@ -1,10 +1,12 @@
 {% for lib in include.data %}
 {% assign api = lib[1] %}
+{% if api.name != "librmm" %}
```

Comment:
Can we instead add a field like `if not api.hidden` and set `hidden: true` in the YAML?

Reply:
Good catch. Fixed. Thanks for that review!